src/utils/hooks/useRouter.tsx

```diff
@@ -34,6 +35,17 @@ interface RouterResult {
   location: any
 }
 
+const replaceHifenKeys = (parsedQuery: { [key: string]: any }) => {
```

Comment:
Also -- we're leaving mapping the values to another PR? (Presumably because the delimiters and date parsing will be more complex?)

Reply:
Yes